tests/test_server.py

```diff
@@ -14,8 +14,11 @@
 """
 
 import sys
+import os
 import time
 
+sys.path.append(os.getcwd())  # noqa
```

Comment:
because I need it to be added before I import anything else from tests, but then, I could treat signals as if they were on the same level....